src/main/java/pl/pszczolkowski/mustdo/domain/task/finder/TasksListSnapshotFinder.java

```diff
@@ -0,0 +1,14 @@
+package pl.pszczolkowski.mustdo.domain.task.finder;
+
+import java.util.List;
+
+import pl.pszczolkowski.mustdo.domain.task.dto.TasksListSnapshot;
+
+public interface TasksListSnapshotFinder {
+
+   TasksListSnapshot findOneById(Long id);
+
+   List<TasksListSnapshot> findAllWithBoardId(Long boardId);
```

Comment:
Uncle Bob says (and I agree with him) that method names in the same context (perfectly in whole project) should be consistent. So it should be "findBy..."